frontend/coprs_frontend/coprs/templates/banner-include.html

```diff
@@ -0,0 +1,14 @@
+<div class="alert alert-danger" role="alert">
+    {% if outage_time and ticket %}
+        <b>Warning!</b> Scheduled outage of Copr servers - {{ outage_time }}. For more information
+        please refer to <a href="https://pagure.io/fedora-infrastructure/issue/{{ ticket }}">this
```

Comment:
Hmm, this is to hardcoded. Especially for people who run their own instance. I would have nothing against it if this file will be in /etc/copr/ and marked as %config

Reply:
yes it is hardcoded. I meant this to be really specific and strict for our release use cases since we do them often and it is a very specific banner (and we want basically every time the same warning message).

what do you mean by marked as %config? what do I achieve from it?

Reply:
When the file is in spec file marked as %config then it can be modified by user and it will not be overwritten during `dnf upgrade` but .rpmnew file will be created instead.